src/corecel/grid/detail/GridAccessor.hh

```diff
@@ -49,19 +58,18 @@ class GridAccessor
 
 //---------------------------------------------------------------------------//
 /*!
- * Helper class for accessing grid data from spans.
+ * Helper class for accessing grid data from a generic grid.
```

Comment:
I think "generic grid" is obsolete (#1616) so perhaps make sure all the stuff in corecel/grid and celeritas/grid say "nonuniform grid".